### PR TITLE
Mytarget Bid Adapter : update adapter to comply with Prebid 5

### PR DESCRIPTION
**Pull request by Denislavrov**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [x] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- test parameters for validating bids
```
{
  bidder: '<bidder name>',
  params: {
    // ...
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [ ] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

**Comment by ChrisHuie** [comment]
@Denislavrov I don't see that advertiserdomain is supported with the meta object. Can you please add. That is now required in 5.0. https://github.com/prebid/Prebid.js/issues/6650

**Comment by Denislavrov** [comment]
> @Denislavrov I don't see that advertiserdomain is supported with the meta object. Can you please add. That is now required in 5.0. #6650

@ChrisHuie, thank you!
I've committed the fixed version

**Comment by ChrisHuie** [comment]
@Denislavrov I just rekicked off the tests we should be good to go on this as soon as you update the imports.

File: modules/mytargetBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
+import * as utils from '../src/utils.js';
```
Comment (ChrisHuie) [inline modules/mytargetBidAdapter.js:1]: Can you please only import the functions needed from utils here? We just went through all adapters this week to eliminate the need for `utils.`

Comment (Denislavrov) [reply modules/mytargetBidAdapter.js]: @ChrisHuie i fixed this import but tests failed again, local test OK!
In First I merge topical master

**Comment by ChrisHuie** [comment]
@Denislavrov this looks good to me

**Pull request merged**